frames/src/main/java/com/checkout/frames/screen/manager/PaymentFormStateManager.kt

```diff
@@ -30,6 +33,17 @@ internal class PaymentFormStateManager(
 
     override val supportedCardSchemeList = provideCardSchemeList()
 
+    override val billingAddressFields = provideBillingFormAddressFields()
+
+    override val mandatoryBillingAddressFields = provideMandatoryBillingAddressFields()
+
+    private fun provideBillingFormAddressFields(): List<AddressField> = billingFormFieldList.ifEmpty {
+        BillingFormFields.fetchAllDefaultBillingFormFields()
+    }
+
+    private fun provideMandatoryBillingAddressFields(): List<AddressField> =
+        BillingFormFields.fetchAllMandatoryBillingFormFields()
```

Comment:
`PaymentStateManager ` was created mostly for data-related things, like phone, address and so on.
Another problem with such implementation is that `billingAddressFields` and `mandatoryBillingAddressFields` will store data for the whole lifecycle of the PaymentForm. But we need it only for billing form and when it's closed GC should be able to collect it.